README.md

```diff
@@ -75,6 +78,7 @@ A string to represent the time when the TestRail run completed in the action.
   uses: q4mobile/stratus-testrail-reporter@v1
   with:
     target_branch: ${{ github.base_ref }}
+    regression_branch: test ## optional
```

Comment:
since the default is `staging`, perhaps readme should reflect

Reply:
I added it above too but it doesn't hurt to have it here either